src/QuickCheckVEngine/RVFI_DII/RVFI.hs

```diff
@@ -460,9 +486,10 @@ rvfiCheck is64 x y assert
 -- | Compare 2 'RVFI_Packet's and produce a 'String' output displaying the
 --   the content of the packet once only for equal inputs or the content of
 --   each input 'RVFI_Packet' if inputs are not succeeding the 'rvfiCheck'
-rvfiCheckAndShow :: Bool -> RVFI_Packet -> RVFI_Packet -> (Maybe Integer) -> (Bool, String)
+rvfiCheckAndShow :: Bool -> Int -> RVFI_Packet -> RVFI_Packet -> (Maybe Integer) -> (Bool, String)
 rvfiCheckAndShow is64 x y assert
```

Comment:
doesn't that need to receive `verbosity` as an argument?

Reply:
oh yeah not sure why it didn't complain